text/entities/0256-entities-data-model.md

```diff
@@ -0,0 +1,790 @@
+
+Entities may be also associated with produced telemetry indirectly, typically by
+virtue of being related to the Producing Entity. For example a Service that produces
+telemetry is also related with a process in which the Service runs, so we say that
+the Service entity is related to the Process entity. The process normally also runs
+on a host, so we say that the Process entity is related to the Host entity.
+
+Note: subsequent OTEPs will define how the Producing Entity is associated with
+traces, metrics and logs and how relations between entities will be specified.
+See [Future Work](#future-work).
+
+The data model below defines a logical model for an entity (irrespective of the physical
+format and encoding of how entity data is recorded).
+
+<table>
+   <tr>
+    <td><strong>Field</strong>
+    </td>
+    <td><strong>Type</strong>
+    </td>
+    <td><strong>Description</strong>
+    </td>
+   </tr>
+   <tr>
+    <td>Type
+    </td>
+    <td>string
+    </td>
+    <td>Defines the type of the Entity. MUST not change during the
+lifetime of the entity. For example: "service" or "host". This field is
+required and MUST not be empty for valid entities.
+    </td>
+   </tr>
+   <tr>
+    <td>Id
+    </td>
+    <td>key/value pair list
+    </td>
+    <td>A set of attributes that identifies the Entity.
+<p>
+MUST not change during the lifetime of the Entity. The Id must contain
+at least one attribute.
+<p>
+Follows OpenTelemetry <a
+href="https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/common#attribute">common
+attribute definition</a>. SHOULD follow OpenTelemetry <a
+href="https://github.com/open-telemetry/semantic-conventions">semantic
+conventions</a> for attributes.
+    </td>
+   </tr>
+   <tr>
+    <td>Attributes
+    </td>
+    <td>key/value pair list
+    </td>
+    <td>A set of descriptive (non-identifying) attributes of the Entity.
+<p>
+MAY change over the lifetime of the entity. MAY be empty. These
+attributes are not part of Entity's identity.
+<p>
+“value” follows <a
+href="https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#type-any">any</a>
+value definition in the OpenTelemetry spec - it can be a scalar value,
+byte array, an array or map of values. Arbitrary deep nesting of values
+for arrays and maps is allowed.
+<p>
+SHOULD follow OpenTelemetry <a
+href="https://github.com/open-telemetry/semantic-conventions">semantic
+conventions</a> for attributes.
+    </td>
+   </tr>
+</table>
+
+### Minimally Sufficient Id
+
+Often a number of attributes of an entity is readily available for the telemetry
+producer to compose an Id from. Of the available attributes the Entity Id should
+include the minimal set of attributes that is sufficient for uniquely identifying
+that entity. No superfluous attributes should be included in the Id set. For example
+a Process on a host can be uniquely identified by `process.id` attribute. Adding for
```

Comment:
We use process as an example in the LID/GID/IDContext discussion as well but define it differently. (PID/StartTime)

I recognize that the example is intentionally simple to illustrate a point, but it seems we should either be consistent across the doc or find another example for this.

Reply:
Good catch, fixed in https://github.com/open-telemetry/oteps/pull/256/commits/a1c99e1da8d2385eeb3ddb09151e236126ec1d85